### DISCUSSION TITLE
参考文献分类不准确

### DISCUSSION THREAD

**Discussion by HanWuDi666** [note286/xduts#211] [Q&A] [answered]
![屏幕截图 2024-11-27 224055](https://github.com/user-attachments/assets/434bc56a-5829-4980-85bb-8c4c297ca5fe)

使用 bib-backend = bibtex，参考文献分类中会多/OL，请问这是为什么

**Comment by note286** [comment, marked as answer]
删除`.bib`文件中的`doi`。

**Comment by HanWuDi666** [comment]
已解决。若.bib文件存在url，还需要删除url

**Comment by epsonlq138** [comment]
根据biblatex-gb7714-2015样式和biblatex宏包使用说明，可采用以下命令去除OL等显示内容
`\usepackage[style=gb7714-2015,doi=false,gbmedium = false,url = false]{biblatex}`
